Formula/rbw.rb

```diff
@@ -0,0 +1,19 @@
+class Rbw < Formula
+  desc "Unoffical Bitwarden CLI client"
+  homepage "https://github.com/doy/rbw"
+  url "https://github.com/doy/rbw/archive/refs/tags/1.3.0.tar.gz"
+  sha256 "7255e322b8c6bd6ee9da00f4c54211e94a132abec15b9f51b8351af31a4744ac"
+  license "MIT"
+  head "#{homepage}.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "pinentry"
+
+  def install
+    system "cargo", "install", *std_cargo_args
```

Comment:
We can probably generate completions here like this:

https://github.com/Homebrew/homebrew-core/blob/12c971dc512d9673fd14ecf740b7df3bc1d1875c/Formula/deno.rb#L73-L78